client/my-sites/people/people-list-item/index.jsx

```diff
@@ -60,11 +64,14 @@ export default React.createClass( {
 				this.props.onRemove &&
 				<div className="people-list-item__actions">
 					<button className="button is-link people-list-item__remove-button" onClick={ this.props.onRemove }>
-						{ this.translate( 'Remove', { context: 'Verb: Remove a user or follower from the blog.' } ) }
+						{ this.props.translate( 'Remove', { context: 'Verb: Remove a user or follower from the blog.' } ) }
```

Comment:
I'd suggest to use no context at all, we have [plenty of existing translations](https://translate.wordpress.com/projects/wpcom/de/default?filters%5Bterm%5D=remove) that would be reused. Apart from that I don't know of a use of the word "Remove" as not a verb.